java/src/gov/usgs/producerclient/ProducerClient.java

```diff
@@ -168,6 +187,12 @@ public static void main(String[] args) {
 			inputDirectory = (String) configJSON.get(INPUT_DIRECTORY);
 			logger.info(
 					"Using configured inputDirectory of: " + inputDirectory);
+
+			// create input directory if it doesn't exist
+			File inputDir = new File(inputDirectory);
```

Comment:
I like directory-self-managing programs, but in this case, do we want to create the input directory?  Seems like maybe that should be the responsibility of whoever is populating that directory, and this might be a good check on someone accidentally putting in the wrong directory name in the config file.  Just a thought...

Reply:
k